Sealnote/src/main/java/com/twistedplane/sealnote/data/Note.java

```diff
@@ -102,12 +101,12 @@ public void writeToParcel(Parcel outParcel, int flags) {
         outParcel.writeInt(mId);
         outParcel.writeInt(mPosition);
         outParcel.writeString(mNoteTitle);
+        outParcel.writeString(mType.name());
```

Comment:
Any reason this was moved from line 110 to this line? If not its better to keep code where it was to make diffs small.